src/views/Home.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="home center">
     <h3 class="accent welcome">{{ subname }}</h3>
-    <Ninja />
+    <!-- <Ninja /> -->
```

Comment:
can we just delete this instead of commenting or just add a comment <-- ad section --> for future sponsor agar koi karega to 😶

Reply:
I kinda agree.